### PR TITLE
Issue #5

### PR DESCRIPTION
**Pull request by tirthpatel7498**
@yash01234 I have changed the variable names. You can check those var names and use them for your table.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -1,8 +1,14 @@
-name: 'Wait'
+name: 'Constable'
 description: 'Create a JavaScript Action grade the project'
+inputs:
```
Comment (dangoslen) [inline action.yml]: Just remove this. I think its throwing an error in the test GH action workflow

**Review by dangoslen — APPROVED** [review]
Changes look good minus my one comment about `inputs:`

File: action.yml (hunk 1 above)
Comment (tirthpatel7498) [reply action.yml]: Done

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -20,4 +20,4 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - uses: ./
+    # - uses: ./
```
Comment (dangoslen) [inline .github/workflows/test.yml]: We shouldn't do this. We need to validate our action works as a black box, not just via unit tests

**Pull request merged**